fairlearn/metrics/true_positive_rate.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+
+from . import MetricsResult
+
+
+def true_positive_rate(y_actual, y_predict, group_id):
```

Comment:
I think this is the equivalent of this scikit learn metric:
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html
the only thing is that it doesn't compute "group_id" metrics like this does

Reply:
sorry recall score, not precision:
https://github.com/scikit-learn/scikit-learn/blob/1495f6924/sklearn/metrics/classification.py#L1577

Reply:
I'm sure - that brings up another point: do we take a dependency on `scikit-learn` for this? It would speed up the initial coding at least, since I would just have to implement the wrapper which groups things up for passing to `scikit-learn`. On the other hand it's another dependency and doing things that way might introduce extra performance bottlenecks. The former concern is probably greater.

Reply:
For now, I'm taking the dependency.....

Reply:
Or maybe not... just doing the 'engine' for now